### PR TITLE
[GDR-2359] Clean up rsd

### PR DESCRIPTION
**Pull request by j-smola**
# Description
## What changed?
Related JIRA issue: [GDR-2359](https://jira.gene.com/jira/browse/GDR-2359)

## Why was it changed?
to remove obsolete datasets and script
to update description of current datasets (

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [x] Package version bumped
- [x] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Review by gladkia — APPROVED** [review]

File: inst/testdata/synthetic_list.yml [hunk 1]
```diff
@@ -1,26 +1,72 @@
...
-- name: finalSE_small
-  file: finalSE_small.RDS
-  description:
-- name: finalSE_invalid_small
-  file: finalSE_invalid_small.RDS
-  description:
-- name: finalSE_combo_with_metrics
-  file: finalSE_combo_with_metrics.RDS
-  description:
+combo_2dose_nonoise:
```
Comment (gladkia) [inline inst/testdata/synthetic_list.yml:1]: 💪

File: README.md [hunk 2]
```diff
@@ -1,19 +1,12 @@
 # gDRtestData
 
-PROJECT DESCRIPTION:
-gDRtestData - R data package with testing dose reponse data
+gDRtestData - R data package with testing dose response data to be used by all gDR packages and in gDR application.
 
-Started on 2020-07-20.
 
-## Directory structure
+# Avialable data
```
Comment (bczech) [inline README.md:6]:
```suggestion
# Available data
```

File: README.md [hunk 3]
```diff
@@ -1,19 +1,12 @@
...
-```
-gDRtestData			→ subdirectory with R teest data package
-	inst/testdata/			→ R datasets provided in the package
-	inst/testdata/rsd			→  raw synthetic datasets
-	inst/annotation_data/		→ annotation data provided in the package
-	man/				→ R package man dir
-	DESCRIPTION			→ R package DESCRIPTION file
-	LICENSE				→ R package license file
-```
+- `inst/testdata/` folder with R datasets provided in the package in qs format
```
Comment (bczech) [inline README.md:8]:
```suggestion
- `inst/testdata/` directory with R datasets provided in the package in qs format
```

File: inst/testdata/synthetic_list.yml [hunk 4]
```diff
@@ -1,26 +1,72 @@
...
+  file: finalMAE_combo_matrix.qs
+  script: generateComboMatrix
+  imported_as_example: TRUE
+  description: "6 x 3 drugs (matrix) for 8 cell lines"
+combo_triple:
+  name: finalMAE_combo_triple
+  file: finalMAE_combo_triple.qs
+  script: generateTripleComboMatrix
+  imported_as_example: TRUE
+  description: "2 x 3 x 2 drugs (few doses) for 4 cell lines"
```
Comment (bczech) [inline inst/testdata/synthetic_list.yml:48]: instead of `few doses` I would put matrix or triple matrix

File: inst/testdata/synthetic_list.yml [hunk 5]
```diff
@@ -1,26 +1,72 @@
...
+medium:
+  name: finalMAE_medium
+  file: finalMAE_medium.qs
+  script: generateMediumData
+  imported_as_example: TRUE
+  description: "40 drugs (6 differents drug_moa) by 15 lines (3 tissues); single agent"
```
Comment (bczech) [inline inst/testdata/synthetic_list.yml:54]:
```suggestion
  description: "40 drugs (6 different drug_moa) by 15 lines (3 tissues); single agent"
```

File: inst/testdata/synthetic_list.yml [hunk 6]
```diff
@@ -1,26 +1,72 @@
...
+small_no_noise:
+  name: finalMAE_small_no_noise
+  file: finalMAE_small_no_noise.qs
+  script: generateNoNoiseRawData
+  imported_as_example: FALSE
+  description: "10 drugs (3 differents drug_moa) by 10 lines (3 tissues); single agent - no noise in the data"
```
Comment (bczech) [inline inst/testdata/synthetic_list.yml:60]:
```suggestion
  description: "10 drugs (3 different drug_moa) by 10 lines (3 tissues); single agent - no noise in the data"
```

File: inst/testdata/synthetic_list.yml [hunk 7]
```diff
@@ -1,26 +1,72 @@
...
+small:
+  name: finalMAE_small
+  file: finalMAE_small.qs
+  script: generateNoiseRawData
+  imported_as_example: TRUE
+  description: "10 drugs (3 differents drug_moa) by 10 lines (3 tissues); single agent"
```
Comment (bczech) [inline inst/testdata/synthetic_list.yml:66]:
```suggestion
  description: "10 drugs (3 different drug_moa) by 10 lines (3 tissues); single agent"
```

File: README.md [hunk 8]
```diff
@@ -1,19 +1,12 @@
...
 
+- `inst/annotation_data/`	annotation data provided in the package
+
+- `inst/scripts/`	scripts to build example dataset
```
Comment (bczech) [inline README.md:12]:
```suggestion
- `inst/scripts/`	scripts to generate example dataset
```

File: inst/testdata/synthetic_list.yml [hunk 9]
```diff
@@ -1,26 +1,72 @@
...
-  description:
-- name: finalSE_combo_with_metrics
-  file: finalSE_combo_with_metrics.RDS
-  description:
+combo_2dose_nonoise:
+  name: finalMAE_combo_2dose_nonoise
+  file: finalMAE_combo_2dose_nonoise.qs
+  script: generateComboNoNoiseData
+  imported_as_example: TRUE
+  description: "3 drugs x 2 co-treatment (1 drug at 2 doses) by 3 lines; co-treatment drug is only as DrugName_2"
```
Comment (bczech) [inline inst/testdata/synthetic_list.yml:6]:
```suggestion
  description: "3 drugs x 2 co-treatment (1 drug at 2 doses) by 3 cell lines; co-treatment drug is only as DrugName_2"
```

File: inst/testdata/synthetic_list.yml [hunk 10]
```diff
@@ -1,26 +1,72 @@
...
+combo_2dose_nonoise2:
+  name: finalMAE_combo_2dose_nonoise2
+  file: finalMAE_combo_2dose_nonoise2.qs
+  script: generateComboNoNoiseData2
+  imported_as_example: FALSE
+  description: "3 drugs x 2 co-treatment (1 drug at 2 doses) by 3 lines; co-treatment drug is also as single agent as DrugName"
```
Comment (bczech) [inline inst/testdata/synthetic_list.yml:12]:
```suggestion
  description: "3 drugs x 2 co-treatment (1 drug at 2 doses) by 3 cell lines; co-treatment drug occurs also as a primary drug"
```

File: inst/testdata/synthetic_list.yml [hunk 11]
```diff
@@ -1,26 +1,72 @@
...
+combo_2dose_nonoise3:
+  name: finalMAE_combo_2dose_nonoise3
+  file: finalMAE_combo_2dose_nonoise3.qs
+  script: generateComboNoNoiseData3
+  imported_as_example: FALSE
+  description: "3 drugs x 2 co-treatment (1 drug at 2 doses) by 3 lines; co-treatment drug does NOT have single agent response"
```
Comment (bczech) [inline inst/testdata/synthetic_list.yml:18]: "co-treatment drug does NOT have single agent response" How does it differ from "cell lines; co-treatment drug is only as DrugName_2"?

**Review by bczech — COMMENTED** [review]
The description of data should be unified to follow the same nomenclature for each dataset

File: README.md (hunk 2 above)
Comment (j-smola) [reply README.md:6]: Done. ✔️

File: README.md (hunk 3 above)
Comment (j-smola) [reply README.md:8]: Done. ✔️

File: inst/testdata/synthetic_list.yml (hunk 4 above)
Comment (j-smola) [reply inst/testdata/synthetic_list.yml:48]: Done. ✔️

File: inst/testdata/synthetic_list.yml (hunk 5 above)
Comment (j-smola) [reply inst/testdata/synthetic_list.yml:54]: Done. ✔️

File: inst/testdata/synthetic_list.yml (hunk 6 above)
Comment (j-smola) [reply inst/testdata/synthetic_list.yml:60]: Done. ✔️

File: inst/testdata/synthetic_list.yml (hunk 7 above)
Comment (j-smola) [reply inst/testdata/synthetic_list.yml:66]: Done. ✔️

File: README.md (hunk 8 above)
Comment (j-smola) [reply README.md:12]: Done. ✔️

File: inst/testdata/synthetic_list.yml (hunk 9 above)
Comment (j-smola) [reply inst/testdata/synthetic_list.yml:6]: Done. ✔️

File: inst/testdata/synthetic_list.yml (hunk 10 above)
Comment (j-smola) [reply inst/testdata/synthetic_list.yml:12]: Done. ✔️

File: inst/testdata/synthetic_list.yml (hunk 11 above)
Comment (j-smola) [reply inst/testdata/synthetic_list.yml:18]: Source of description https://github.com/gdrplatform/gDRtestData/blob/master/inst/scripts/generate_example_data.R#L17-L18

@bczech  How to improve that?

**Review by bczech — APPROVED** [review]

**Pull request merged**